lib/ColorThief/Image/Adapter/ImagickImageAdapter.php

```diff
@@ -83,10 +101,10 @@ public function getPixelColor($x, $y)
         // So we ask for normalized values, and then we un-normalize it ourselves.
         $colorArray = $pixel->getColor(true);
         $color = new \stdClass();
-        $color->red = round($colorArray['r'] * 255);
-        $color->green = round($colorArray['g'] * 255);
-        $color->blue = round($colorArray['b'] * 255);
-        $color->alpha = 127 - round($colorArray['a'] * 127);
+        $color->red = (int)round($colorArray['r'] * 255);
+        $color->green = (int)round($colorArray['g'] * 255);
+        $color->blue = (int)round($colorArray['b'] * 255);
+        $color->alpha = (int)(127 - round($colorArray['a'] * 127));
```

Comment:
It's a bit odd for me that the cast wasn't required before this change, but anyway it's the right way to do, as it's already done [in the GMagick adapter](https://github.com/ksubileau/color-thief-php/blob/491ccffc5dbcc76df381c999817c4ae2d67a4ffb/lib/ColorThief/Image/Adapter/GmagickImageAdapter.php#L98).

Reply:
Actually the cast is still not required (the type inconsistency is inconsequential thanks to PHP's type juggling), but I took the liberty to correct it anyway.